utils.js

```diff
@@ -479,7 +480,14 @@ export function adjustLicenseInformation(licenses) {
   });
   if (expressions.length >= 1) {
     if (expressions.length > 1) {
-      console.warn("multiple license expressions found", expressions);
+      return [
+        {
+          expression: expressions
+            .map((e) => e.expression || "")
+            .filter(Boolean)
+            .join(" OR "),
```

Comment:
Let's really hope that some of those expressions do not include AND. Don't think we can add brackets to specify operator precedence 😉